tests/test_data/sii-dte/DTE--96670340-7--61--110616--cleaned.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version='1.0' encoding='ISO-8859-1'?>
```

Comment:
If this is the cleaned file, shouldn't it have fixed the missing newline at the end? I don't remember what we did for the other "cleaned" files